CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.35.1]
+### Changed
+- Allow quick Add Account name based on default name ([#20168](https://github.com/MetaMask/metamask-extension/pull/20168))
+- Store default gas settings by network ([#20576](https://github.com/MetaMask/metamask-extension/pull/20576), [#20632](https://github.com/MetaMask/metamask-extension/pull/20632))
```

Comment:
#20632 hasn't been included in the RC yet, but it was listed here preemptively since it is a release blocker

Reply:
It has been included now